_pages/production.md

```diff
@@ -54,8 +54,9 @@ If you wish to deploy an application to our staging environment, then create a 
 
 Before you can deploy your application to the production environment, you will need to create a separate app on our dashboard that contains your production certificate, urls and logo. Here are the steps to complete your production configuration app:
 1. Create a new app on the dashboard <https://dashboard.int.identitysandbox.gov/>
-2. Enter the name of the application **as it appears in your IAA** in the "App Name" field
-3. Enter the name of the application **as you want it to appear to users** in the "Friendly Name" field
+2. Indicate that the app configuration is for a Production application 
+2. Enter the name of the application **as it appears in your IAA** in the "App Name" field - Do **NOT** use environment names such as "Prod" or "Production"
+3. Enter the name of the application **as you want it to appear to users** in the "Friendly Name" field - Do **NOT** use environment names such as "Prod" or "Production"
```

Comment:
Could you be more specific? 🤡

Reply:
**question:** one thought - what if they _want_ the environment name included in what is presented to users?